integration-tests/rpc-nodes.spec.ts

```diff
@@ -20,7 +20,7 @@ CONFIGS().forEach(
   }) => {
     const Tezos = lib;
 
-    const jakartanet = protocol === Protocols.PtJakart2 ? test : test.skip;
+    const limanet = protocol === Protocols.PtLimaPtL ? test : test.skip;
     const kathmandunetAndAlpha = protocol === Protocols.PtKathman || protocol === Protocols.ProtoALpha ? test : test.skip;
```

Comment:
`kathmandunetAndAlpha` constant can be removed, and the tests changed to `it`. The behavior should now be the same on each testnets